src/fsharp/NicePrint.fs

```diff
@@ -1198,7 +1198,18 @@ module private PrintTypes =
 
     let prettyLayoutOfTypeNoConstraints denv ty = 
         let ty, _cxs = PrettyTypes.PrettifyType denv.g ty
-        layoutTypeWithInfoAndPrec denv SimplifyTypes.typeSimplificationInfo0 5 ty 
+        layoutTypeWithInfoAndPrec denv SimplifyTypes.typeSimplificationInfo0 5 ty
+
+    let prettyLayoutOfValReturnType denv (v: ValRef) =
```

Comment:
This is not a "pretty" function since it never calls `PrettyTypes.PrettifyType` of an associated routine.